src/vs/platform/request/common/request.ts

```diff
@@ -155,6 +155,12 @@ function registerProxyConfigurations(scope: ConfigurationScope): void {
 				markdownDescription: localize('proxyKerberosServicePrincipal', "Overrides the principal service name for Kerberos authentication with the HTTP proxy. A default based on the proxy hostname is used when this is not set."),
 				restricted: true
 			},
+			'http.noProxy': {
+				type: 'array',
+				items: { type: 'string' },
+				markdownDescription: localize('noProxy', "The noProxy variable specifies domain names, separated by commas, for which proxy settings should be ignored for HTTP/HTTPS requests made by extensions."),
```

Comment:
This can be shorter now: "Specifies domain names for which proxy settings should be ignored for HTTP/HTTPS requests."

Reply:
Thanks for the info. I have updated the PR with the updated description. https://github.com/microsoft/vscode/blob/d5b121787ac56b507385fdb81e002fc938d67f5d/src/vs/platform/request/common/request.ts#L158-L163